README.md

```diff
@@ -8,27 +8,29 @@ The original datasets and documents are sourced from the Chicago Police Departme
 
 ### 2. Research Question
 
-With our dataset we would look answer the main inferential question of whether an officer's salary influences their complaint rate (numbe of complaints per year) using a subset of the data from the years 2005 to 2015.
+With our dataset we would look answer the main inferential question of whether an police staff member's salary influences their complaint rate (number of complaints per year) using a subset of the data from the years 2005 to 2015. We suspect a relationship between these two variables 
 
 ### 3. Analysis Plan
 
-As the CPD data to answer our research question is stored in 3 separate files (i.e. one file for salaries, and two filed related to information about the complaints lodged against officers), we will have to consolidate these files by joining them through the Unique Officer IDs. The first step in our analysis plan is merging and cleaning the dataframes. At this stage, we will also calculate the complaint rate (number of complaints per year) for each officer. Once we have the our data cleaned, we will use a linear regression model to determine whether officer's salary can be used to explain the difference in officer complaint rate. Other variables that may impact on complaint rate include officer age, gender and race; therefore, we will consider including these variables to our linear model. Our analysis plan will evolve as our knowledge of linear regression increases through the DSCI 561 course.    
+As the data to answer our research question is stored in 3 separate files (i.e. one file for salaries, and two files related to information about the complaints lodged against police staff members), we will have to consolidate these files by joining them through the Unique ID field. The first step in our analysis plan is merging and cleaning the dataframes. At this stage, we will also calculate the complaint rate (number of complaints per year per member) for each police staff member. Once we have cleaned our data, we will use a linear regression model to determine whether a police staff member's salary is associated with their complaint rate. The response variable for our model is the complaint rate while the explanatory variable is the salary. We have also considered other variables that may impact on complaint rate which include the staff member's age, gender and race; therefore, we will consider including these variables to our linear model if time permits. Our analysis plan may evolve as our knowledge of linear regression increases through the DSCI 561 course.    
```

Comment:
"At this stage, we will also calculate the complaint rate (number of complaints per year per member) for each police staff 
member."  
This calculation was actually done in a separate dataframe, prior to creating the visualization.   We probably should add it as variable to the merged df, will incorporate next week.

Reply:
Great idea, I will create a Milestone 2 project and add it there.